Src/Processor/CURL.cpp

```diff
@@ -3,24 +3,26 @@
 #include "UUID.h"
 #include "CURL.h"
 
+PP_NAMESPACE_BEGIN
+
 size_t EmptyCURLWriteData(void *buffer, size_t size, size_t nmemb, void *userp)
 {
 	return size * nmemb;
 }
 
-CCURL::CCURL()
+CURL::CURL()
 {
 	_pCURL = curl_easy_init();
 	curl_easy_setopt(_pCURL, CURLOPT_SSL_VERIFYPEER, false);
 	curl_easy_setopt(_pCURL, CURLOPT_WRITEFUNCTION, EmptyCURLWriteData);
 }
 
-CCURL::~CCURL()
+CURL::~CURL()
 {
 	curl_easy_cleanup(_pCURL);
```

Comment:
a unique_ptr with a custom destructor could be good

Reply:
Good point! I didn't even know `unique_ptr` supports custom deleters.